### DISCUSSION TITLE
Anfänger

### DISCUSSION THREAD

**Discussion by hotschy** [RPiList/specials#1345] [General]
Hallo,
nutze Pi-Hole neu und hab deshalb vielleicht eine etwas Blöde Frage.
Suche nach einer Adlist zur Kindersicherung. Hier finde ich Blocklisten aber wie kann ich die einbinden?

**Comment by RPiList** [comment]
https://www.youtube.com/watch?v=Aisa8qAjllc

hier wird u.a. die Möglichkeit des Einbindens über Teleporter-Datei gezeigt.

Die Teleporter Datei gibts dann hier: https://github.com/RPiList/specials/blob/master/teleporter_ohne-Banken_2023-07.tar.gz

**Reply by hotschy** [reply]
Merci!
